### PR TITLE
Run formatter on lib/logger/test/logger/translator_test.exs.

### PR DESCRIPTION
**Pull request by fmcgeough**
Formatter moved indentation on a lot of lines to align with
capture_log instead of assert.

### CODE REVIEW

File: lib/logger/test/logger/translator_test.exs [hunk 1]
```diff
@@ -107,601 +108,619 @@ defmodule Logger.TranslatorTest do
...
   test "translates :proc_lib crashes with name" do
-    {:ok, pid} = Task.start_link(__MODULE__, :task,
-                                 [self(), fn() ->
-                                            Process.register(self(), __MODULE__)
-                                            raise "oops"
-                                          end])
+    {:ok, pid} =
+      Task.start_link(__MODULE__, :task, [
+        self(),
+        fn ->
```
Comment (josevalim) [inline lib/logger/test/logger/translator_test.exs]: Please extract this function out to a variable. :)

File: lib/logger/test/logger/translator_test.exs [hunk 2]
```diff
@@ -107,601 +108,619 @@ defmodule Logger.TranslatorTest do
...
   test "translates :proc_lib crashes without initial call" do
-    {:ok, pid} = Task.start_link(__MODULE__, :task,
-                                 [self(), fn() ->
-                                            Process.delete(:"$initial_call")
-                                            raise "oops"
-                                          end])
+    {:ok, pid} =
+      Task.start_link(__MODULE__, :task, [
+        self(),
+        fn ->
```
Comment (josevalim) [inline lib/logger/test/logger/translator_test.exs]: Same here.

File: lib/logger/test/logger/translator_test.exs [hunk 3]
```diff
@@ -107,601 +108,619 @@ defmodule Logger.TranslatorTest do
...
   test "translates :proc_lib crashes with neighbour with name" do
-    {:ok, pid} = Task.start_link(__MODULE__, :sub_task,
-                                 [self(), fn(pid2) ->
-                                            Process.register(pid2, __MODULE__)
-                                            raise "oops"
-                                          end])
+    {:ok, pid} =
+      Task.start_link(__MODULE__, :sub_task, [
+        self(),
+        fn pid2 ->
```
Comment (josevalim) [inline lib/logger/test/logger/translator_test.exs]: Same here.

**Comment by josevalim** [comment]
@fmcgeough some tiny comments and we should be good to go. Thank you!

File: lib/logger/test/logger/translator_test.exs (hunk 1 above)
Comment (fmcgeough) [reply lib/logger/test/logger/translator_test.exs]: I see what you mean. the Task.start_link is a bit harder to read than necessary.

File: lib/logger/test/logger/translator_test.exs (hunk 3 above)
Comment (fmcgeough) [reply lib/logger/test/logger/translator_test.exs]: Yep. I think I got them all.

File: lib/logger/test/logger/translator_test.exs (hunk 2 above)
Comment (fmcgeough) [reply lib/logger/test/logger/translator_test.exs]: got it

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**